### ISSUE TITLE
[Discussion] Changes for bullet protection

### ISSUE THREAD

**Issue by Arzio** [nexusnode/crafting-dead#102]
I had an idea, and I will explain it by steps:

1 - Make all helmets and vests always have a bullet protection level that can be between I and IV. That level range can be set per item (for example, Riot Vest's range is from II to IV, normal vest's ranges are from I to III). That protection level is set to a random level within that range when the item spawns (even when you get from creative inventory), so it is not a level that is fixed, it is always random based on a range per Vest instance. The more the protection level, the rarest is to get it.

You can implement the way you want it, because the idea is to just provide protection levels.

2 - Make all guns ignore 100% of Minecraft's armor slots, but not ignore CD's vests and helmets. Only vests and helmets will protect the player against guns, so the server doesnt care if you are wearing a diamond armor. Vests can only protect bullet damage (and maybe melee damage), helmets can only protect against bullet damage on head. Also make clothes not protect against bullet damage, but protect against infection and bleeding.

So basically why that 2nd idea?
It would be great to differentiate CD's vests and helmets from Minecraft's armors, because we would make the players search for vests and helmets even if they have a Netherite set (or better), for example. Players would not only need to equip vests and helmets, but also good Minecraft armors, because Crafting Dead's melees, unlike guns, are affected by Minecraft armors.

The protection against Infection and Bleeding of the clothes could have levels aswell.

This is just an idea so it's not final, feel free to discuss and change the idea <3
